app/src/main/java/com/h/pixeldroid/fragments/feeds/FeedFragment.kt

```diff
@@ -72,7 +76,13 @@ open class FeedFragment<T: FeedContent, VH: RecyclerView.ViewHolder?>: Fragment(
 
         swipeRefreshLayout.setOnRefreshListener {
             //by invalidating data, loadInitial will be called again
-            factory.liveData.value!!.invalidate()
+            if (Utils.hasInternet(requireContext())) {
+                factory.liveData.value!!.invalidate()
+            } else {
+                startActivity(Intent(requireContext(), MainActivity::class.java).apply {
+                    flags = Intent.FLAG_ACTIVITY_NEW_TASK
```

Comment:
On peut toujours revenir en arrière, faudrait ajouter `FLAG_ACTIVITY_CLEAR_TASK`

Reply:
nah ça fonctionne pas non plus

Reply:
Non, je dis les deux: `FLAG_ACTIVITY_NEW_TASK or FLAG_ACTIVITY_CLEAR_TASK` (c'est un ou sur le string binaire, ça va les combiner)

Reply:
still not

Reply:
ah wait je suis débile je crois

Reply:
Possible, mais je pense pas, j'ai essayé de faire comme je fais autre part (et où ça marche bien comme il faut) mais c'est pareil:

val intent = Intent(requireActivity(), MainActivity::class.java)
intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
startActivity(intent)
```